tests/integration.rs

```diff
@@ -120,6 +120,37 @@ fn invert() -> Result<()> {
   assert_output_eq(&["resize:1:1", "all"], "1")
 }
 
+#[test]
+fn save() -> Result<()> {
+  let path = "output.png";
+
+  assert_output_eq(
+    &["resize:4:4", &format!("save:{}", path), "top"],
+    "1111
+     1111
+     0000
+     0000",
+  )?;
+
+  let image = image::open(path)?;
+  for pixel in image.as_rgb8().unwrap().pixels() {
+    assert_eq!(*pixel, image::Rgb::<u8>([0, 0, 0]));
+  }
```

Comment:
Maybe we should just get the pixels as a vector, and assert_eq the contents, as well as the width and the height.

Reply:
My most recent commit attempts to do this, still unsure if there's a better way of doing it. Let me know what you think